aea/launcher.py

```diff
@@ -173,7 +173,7 @@ def create_async_task(self, loop: AbstractEventLoop) -> TaskAwaitable:
             raise ValueError(
                 "Agent runtime is not async compatible. Please use runtime_mode=async"
             )
-        return loop.create_task(self._agent.runtime.start_and_wait_completed())
+        return loop.create_task(self._agent.runtime.start_and_wait_completed())  # type: ignore
```

Comment:
@solarw for you to investigate. I just added the ignore temporarily. It is masking an issue!